proxy.php

```diff
@@ -260,6 +299,14 @@ function getHttpContentAndStatus($url, $timeout, $user_agent)
             curl_setopt($ch, CURLOPT_POSTFIELDS, $stream_options['http']['content']);
         }
 
+        if (isset($stream_options['ssl']['verify_peer']) && $stream_options['ssl']['verify_peer'] == false) {
```

Comment:
is this the same as using `DEBUG_PROXY` in the `if`?

Reply:
No, that's done above, this translates the stream options to curl options.

Reply:
Not sure I'm getting it. By the looks they are only set to false when using `$DEBUG_PROXY = true` correct? I'm thinking instead of translating it might be easier to read by using `if($DEBUG_PROXY)`. May be also less fragile if eg for some reason the stream option changes it doesn't change the curl behaviour etc. But I didn't have a deep look I might be missing something.

Reply:
The existing code follows the pattern of:

1. set stream_options for file_get_contents
2. if curl, transform already set stream options to curl options

The stream option name won't change, it's the actual php option name. It might change in a PHP version, but that's different than a developer deciding to rename it (which they can't do).